gph/python/ripser_interface.py

```diff
@@ -447,7 +459,16 @@ def ripser_parallel(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
     if metric == 'precomputed':
         dm = X
     else:
-        dm = pairwise_distances(X, metric=metric, **metric_params)
+        if not issparse(X) and thresh != np.inf and \
+                metric in ['euclidean', 'minkowski']:
+            if metric == 'euclidean':
+                p = 2
+            elif metric == 'minkowski':
+                p = metric_params['p']
```

Comment:
I think this should be `p = metric_params.get('p', 2)` because `minkowski` has default `2` e.g. in `pdist`.

Reply:
You are right !

Reply:
Done